docs/how-to-guides/convert-changelog-to-table.md

```diff
@@ -10,113 +10,139 @@ keywords: changelog, table, materialize
-|K                                                      |V1                                                     |V2                                                     |V3                                                     |
-+-------------------------------------------------------+-------------------------------------------------------+-------------------------------------------------------+-------------------------------------------------------+
-|k1                                                     |2                                                      |c                                                      |false                                                  |
-
-
-SELECT k, v1, v2, v3 FROM t1 WHERE k='k2';
-+-------------------------------------------------------+-------------------------------------------------------+-------------------------------------------------------+-------------------------------------------------------+
-|K                                                      |V1                                                     |V2                                                     |V3                                                     |
-+-------------------------------------------------------+-------------------------------------------------------+-------------------------------------------------------+-------------------------------------------------------+
-|k2                                                     |4                                                      |e                                                      |true                                                   |
-
-
-SELECT k, v1, v2, v3 FROM t1 WHERE k='k3';
-+-------------------------------------------------------+-------------------------------------------------------+-------------------------------------------------------+-------------------------------------------------------+
-|K                                                      |V1                                                     |V2                                                     |V3                                                     |
-+-------------------------------------------------------+-------------------------------------------------------+-------------------------------------------------------+-------------------------------------------------------+
-|k3                                                     |3                                                      |d                                                      |true                                                   |
+SELECT * FROM latest_events;
++------+------+------+------+
+|K     |V1    |V2    |V3    |
++------+------+------+------+
+|k1    |2     |c     |false |
+|k2    |4     |e     |true  |
+|k3    |3     |d     |true  |
 ```
+
+Notice how for each key, the columns reflect the latest set of values.
```

Comment:
It might also be good to explain, that `LATEST_BY_OFFSET` does not support deletes / tombstones, because it's an aggregation and thus "delete semantics" do not apply.